distill.py

```diff
@@ -3,98 +3,128 @@
 
 RDF = ROOT.ROOT.RDataFrame
 
-# Extracted from: DS1/block1/input_files.h
+# Due to fact that not all files from 'prefix' are used in original analysis 
+# the desired ones are specified in the input_files_DS1.txt
+source_file       = "input_files_DS1.txt"
 input_ntuple_name = "TotemNtuple"
 prefix            = "/eos/totem/data/cmstotem/2015/90m/Totem/Ntuple/version2/4495/"
-input_files       = prefix + "Totem*" # Expands to http://cern.ch/go/RT7v
+input_files       = [prefix + line.rstrip('\n') for line in open(source_file)]
 
-# Branches clasified by diagonal
-diagonals = {
-    # return a tuple: ([left] verticals in 45, [right] verticals in 56))
-    "d45b_56t"  : (["track_rp_5",   "track_rp_21",  "track_rp_25"],
-                   ["track_rp_104", "track_rp_120", "track_rp_124"]),
-    "ad45b_56b" : (["track_rp_5",   "track_rp_21",  "track_rp_25"],
-                   ["track_rp_104", "track_rp_120", "track_rp_124"]),
-    "d45t_56b"  : (["track_rp_4",   "track_rp_20",  "track_rp_24"],
-                   ["track_rp_105", "track_rp_121", "track_rp_125"]),
-    "ad45t_56t" : (["track_rp_4",   "track_rp_20",  "track_rp_24"],
-                   ["track_rp_105", "track_rp_121", "track_rp_125"])
-}
+# Convert to PyROOT vector
```

Comment:
Although I think your approach is more convenient for an interactive execution in a Python Notebook,  I think it's better to follow the current approach (using one dictionary with all diagonals and tracks). 

One reason is that we want to produce the same result as `distill.cc` where we run one single diagonal per execution. So I will add here a parser to accept arguments where we can reproduce the options from `distill.cc`.

In summary, could you please modify `distill.py` just to consider the new `source_file`?

Reply:
Sure, the approach from the commit does not make sense in a script file at all, I thought you wanted it to be committed just for future reference. In a moment I will push the distill.py only with the new source_file